invenio_rdm_records/resources/serializers/csl/__init__.py

```diff
@@ -51,6 +50,29 @@ def _clean_result(text):
     return _clean_result(str(bib.bibliography()[0]))
 
 
+def get_style_location(style, locale):
+    """Parse style and locale.
+
+    Argument location precedence: kwargs > view_args > query
+    """
+    _default_style = StringCitationSerializer._default_style
+    _default_locale = StringCitationSerializer._default_locale
+
+    csl_args = {
+        "style": style if style else _default_style,
+        "locale": locale if locale else _default_locale
+    }
+    try:
+        csl_args["style"] = get_style_filepath(csl_args["style"].lower())
+    except StyleNotFoundError:
```

Comment:
Shelved:  Instead of catching the error here, you can provide the error mapping (see ``error_handlers`` in  https://flask-resources.readthedocs.io/en/latest/usage.html#larger-example)

Reply:
We will take this comment in the next PR: https://github.com/inveniosoftware/invenio-rdm-records/pull/682
We will merge this one so that it will not block others :)